source/connect/advanced-connection.txt

```diff
@@ -12,3 +12,88 @@ Advanced Connection Tab
    :depth: 1
    :class: singlecol
 
+
+The :guilabel:`Advanced` connection tab provides additional connection options 
+for your deployment. These options can be used alongside the 
+:ref:`General <general-connection-tab>`, :ref:`Authentication 
+<authentication-connection-tab>`, :ref:`TLS / SSL <tls-ssl-tab>`, and 
+:ref:`Proxy / SSH Tunnel <ssh-connection>` connection options. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Advanced` tab.
+
+      Select a :guilabel:`Read Preference` from the following options:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50 
+
+         * - Read Preference
+           - Description
+
+         * - primary
+           - Default mode. All operations read from the current replica set primary.
+
+         * - primaryPreferred
+           - In most situations, operations read from the primary but if it is 
+             unavailable, operations read from secondary members. 
```

Comment:
Consider omitting "in most situations". It's a bit vague. I think we can say something like:

"Operations read from the primary unless the primary is unavailable. If the primary is unavailable, operations read from secondary members".

Same idea applies to the description of Secondary Preferred.